apps/login/locales/en.json

```
@@ -8,6 +8,16 @@
     "addAnother": "Add another account",
     "noResults": "No accounts found"
   },
+  "logout": {
+    "title": "Logout",
+    "description": "Click the accounts you want to clear",
```

```suggestion
    "description": "Click an account to end the session",
```

I think the word "clear" is a bit too ambiguous